Loop/Managers/LoopDataManager.swift

```diff
@@ -1156,6 +1157,10 @@ extension LoopDataManager {
             throw LoopError.glucoseTooOld(date: glucose.startDate)
         }
 
+        guard lastGlucoseDate.timeIntervalSince(now()) <= LoopCoreConstants.inputDataRecencyInterval else {
```

Comment:
I think this should use a different constant, as it's not checking "recency", and I'm not sure 15 minutes is right cutoff. The only future values we should allow are probably due to device clock drift, and that's probably a smaller window that 15 minutes.

Reply:
I agree - I stuck it in as a placeholder.
Does LoopCoreConstants.futureGlucoseDataInterval match the naming protocol?
I'll put in 5 minutes.